src/Compilers/Core/Portable/SourceGeneration/GeneratorDriver.cs

```diff
@@ -34,6 +34,11 @@ internal GeneratorDriver(GeneratorDriverState state)
 
         internal GeneratorDriver(ParseOptions parseOptions, ImmutableArray<ISourceGenerator> generators, AnalyzerConfigOptionsProvider optionsProvider, ImmutableArray<AdditionalText> additionalTexts)
         {
+            // PROTOTYPE: we currently drop any incremental generators.
```

Comment:
nit: typically `// PROTOTYPE` comments include the feature name, e.g. `// PROTOTYPE(source-generators)`